x-pack/plugins/canvas/public/components/sidebar/sidebar_content.js

```diff
@@ -21,22 +24,43 @@ const branches = [
   // no elements selected
   branch(
     ({ selectedToplevelNodes }) => !selectedToplevelNodes.length,
-    renderComponent(GlobalConfig)
+    renderComponent(() => (
+      <>
```

Comment:
What is this?  This should likely be `<Fragment />`... empty tags like this are non-standard.

Reply:
The empty brackets `<></>` syntax is shorthand for `React.Fragment` introduced in 16.2: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#support-for-fragment-syntax. 

I was trying something differently and thought it was just a smidge quicker than explicitly using `Fragment`, but I'm not committed to one syntax or the other. I can use `<Fragment>` instead of `<>` if that would improve readability.